src/Xamarin.Android.Tools.ApiXmlAdjuster/JavaApi.XmlModel.cs

```diff
@@ -41,30 +42,30 @@ public override string ToString ()
 
 	public abstract partial class JavaType
 	{
-		protected JavaType (JavaPackage parent)
+		protected JavaType (JavaPackage? parent)
 		{
 			Parent = parent;
 			
 			Implements = new List<JavaImplements> ();
 			Members = new List<JavaMember> ();
 		}
 
-		public JavaPackage Parent { get; private set; }
+		public  JavaPackage?            Parent                  { get; private set; }
 
-		public bool IsReferenceOnly { get; set; }
+		public  bool                    IsReferenceOnly         { get; set; }
 
-		public bool Abstract { get; set; }
-		public string Deprecated { get; set; }
-		public bool Final { get; set; }
-		public string Name { get; set; }
-		public bool Static { get; set; }
-		public string Visibility { get; set; }
+		public  bool                    Abstract                { get; set; }
+		public  string?                 Deprecated              { get; set; }
+		public  bool                    Final                   { get; set; }
+		public  string?                 Name                    { get; set; }
+		public  bool                    Static                  { get; set; }
+		public  string?                 Visibility              { get; set; }
```

Comment:
We shouldn't be indenting things like this.  This isn't supported by code formatters such as `.editorconfig` and IDEs, so it causes a constant tooling fight for most people.

Reply:
But it's so much uglier to easily read & scan without the extra whitespace. :-(